### PR TITLE
📟 🎨 Theme for # daz set ~$

### PR DESCRIPTION
**Pull request by VGamezz19**
Hey! i've do a new theme for command `set`!  I follow this patterns:

```javascript
// _________________________________________________
// #title,description,tags,written      --> Red     |     
// #Old content                         --> Yellow  |
// #User Input text                     --> White   |
// #PathFolders                         --> Cyan    |
// #________________________________________________|
```

bay the way, i saw that you don't refactor your `new Date` for `info.modified` & `info.created`
in lane `48` there is a commented date refactored 
```javascript
 //localDate = `${localDate.getFullYear()}-${localDate.getMonth() + 1}-${localDate.getDate()}:${localDate.getHours()}:${localDate.getMinutes()}:${localDate.getSeconds()}`
```
this code should show `2018-2-5-9:33:41`

I didn't put it in the PR becouse i don't know if you prefer put the full date stamp.

⚠️ Please let me to know if you like this theme!

### CODE REVIEW

**Comment by didaquis** [comment]
I confirm code of this pull request run perfectly.

File: src/set.js [hunk 1]
```diff
@@ -41,10 +44,12 @@ function set(basePath, input, output, terminal) {
...
 
         if (step > 3) {
+            let localDate = new Date()
+            //localDate = `${localDate.getFullYear()}-${localDate.getMonth() + 1}-${localDate.getDate()}:${localDate.getHours()}:${localDate.getMinutes()}:${localDate.getSeconds()}`
             if (!info.created)
-                info.created = new Date()
+                info.created = localDate
             else
-                info.modified = new Date()
+                info.modified = localDate
```
Comment (manuelbarzi) [inline src/set.js]: WARN! these dates are part of the output file, not the terminal, so it should not be wrapped with colouring code

File: src/set.js [hunk 2]
```diff
@@ -10,11 +11,13 @@ function set(basePath, input, output, terminal) {
...
 
-    const info = fs.existsSync(file) ? JSON.parse(fs.readFileSync(file, 'utf-8')) : { title: folder }
+    const info = fs.existsSync(file) ? JSON.parse(fs.readFileSync(file, 'utf-8')) : {
+        title: folder
+    }
 
     let step = 1
 
-    output.write(`title [${info.title}]: `)
+    output.write(`\n${color.Color_Off}${color.Blue}# ${color.Color_Off}${color.Red}title${color.Color_Off} [${color.Yellow}${info.title}${color.Color_Off}] ~$ `)
```
Comment (manuelbarzi) [inline src/set.js]: MAY be more convenient to provide a helper script for colouring, instead of doing it directly here

**Review by manuelbarzi — COMMENTED** [review]
NICE! just a few reviews...

**Comment by VGamezz19** [comment]
hey!!! @manuelbarzi, as we discussed, I have separated the responsibilities and theme methods are located in `theme/set` please check if files, functions, etc,  names are corrects, structure and implementation

- tree `/src`

```sh
.
├── Daz.js
├── DazCli.js
├── array-polyfills.js
├── find.js
├── get.js
├── help.js
├── index.js
├── set.js
└── theme
    ├── colors.js
    ├── index.js
    └── set.js
```

**Comment by manuelbarzi** [comment]
thank you, @VGamezz19! your contributions are already merged ,)
thank you, @didaquis, for reviewing!

**Pull request merged**